Tests/conf.json

```diff
@@ -20,6 +20,18 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
+        {
+            "playbookID": "Test Playbook - NGFW Scan",
+			"integrations": [
+				"VirusTotal (API v3)",
+				"Cortex XDR - IR"
```

Comment:
why you need cortex XDR IR integration for XSIAM playbook?
same for the instance name

Reply:
removed